src/components/Search.tsx

```diff
@@ -36,9 +91,9 @@ function Search({query, policyIDs}: SearchProps) {
             return;
         }
 
-        SearchActions.search(hash, query, 0, policyIDs);
+        SearchActions.search({hash, query, policyIDs, offset: 0, sortBy, sortOrder});
```

Comment:
I wonder if we should be computing a new hash for each set of sorting param since the search results would be different than what we currently have. I think we should at least reset the offset, since the following seems to be possible:

1. Open the search page - the default sort for date desc applies
2. Scroll down to load more results, offset will be updated to 50
3. Sort by a different column, e.g. category
4. We'll fetch the results from the API sorted by category, but we'll get the 2nd "page" since the offset is 50.

Reply:
Maybe one optimization would be when `hasMoreResults = false` since we know we have all available data and wouldn't need to fetch anything else to properly sort things in the client.

Reply:
> I wonder if we should be computing a new hash for each set of sorting param

I agree with this and I will add sortBy and sortOrder to hash generation.

However I think the scenario you described would not happen because in this specific line offset is set always to `0` inside this use effect.
So in your scenario in point 3 if you sort by different column or change order, then this `useEffect` block will be in place and a fresh call will be made with `offset: 0`. If you keep scrolling then another call will be made with offset 50 or whatever, but the one with the 0 was already there.

Perhaps another solution is to scroll list to top if you switch sort/order?